diff -Nru old\net\java\xades\security\xml\XAdES\ArchivalXAdESImpl.java new\net\java\xades\security\xml\XAdES\ArchivalXAdESImpl.java
--- old\net\java\xades\security\xml\XAdES\ArchivalXAdESImpl.java	Mon Jun 07 11:44:18 2010
+++ new\net\java\xades\security\xml\XAdES\ArchivalXAdESImpl.java	Mon Sep 13 18:17:20 2010
@@ -1,6 +1,9 @@
 package net.java.xades.security.xml.XAdES;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.TimeZone;
 
 import javax.xml.crypto.MarshalException;
 
diff -Nru old\net\java\xades\security\xml\XAdES\BasicXAdES.java new\net\java\xades\security\xml\XAdES\BasicXAdES.java
--- old\net\java\xades\security\xml\XAdES\BasicXAdES.java	Thu Jun 10 11:34:16 2010
+++ new\net\java\xades\security\xml\XAdES\BasicXAdES.java	Wed Jun 23 15:05:18 2010
@@ -167,7 +167,7 @@
         List transforms = null;
 
         String spId = sp.getId();
-        Reference spReference = getReference(spId, transforms, xadesNamespace + "SignedProperties");
+        Reference spReference = getReference(spId, transforms, "http://uri.etsi.org/01903#" + "SignedProperties");
         referencesIdList.add(spReference);
 
         super.sign(certificate, privateKey, signatureMethod, referencesIdList, signatureIdPrefix,
diff -Nru old\net\java\xades\security\xml\XAdES\DataObjectFormatDetails.java new\net\java\xades\security\xml\XAdES\DataObjectFormatDetails.java
--- old\net\java\xades\security\xml\XAdES\DataObjectFormatDetails.java	Mon Jun 07 11:44:18 2010
+++ new\net\java\xades\security\xml\XAdES\DataObjectFormatDetails.java	Mon Sep 27 12:37:24 2010
@@ -29,7 +29,11 @@
 
         Element description = createElement("Description");
         description.setTextContent(dataObjectFormat.getDescription());
-
+        
+        // Se agrega la sentencia para ajustarnos al estandar
+        // El nodo "Description" debe aparecer antes que el nodo "ObjectIdentifier"
+        getNode().appendChild(description);
+        
         ObjectIdentifier objectIdentifier = dataObjectFormat.getObjectIdentifier();
 
         if (objectIdentifier != null)
@@ -46,7 +50,11 @@
         // TODO: must ensure that there is an ObjectReference attribute, otherwise an Exception
         // should be raised
         setAttribute("ObjectReference", dataObjectFormat.getObjectReference());
-        getNode().appendChild(description);
+
+        // Se elimina la sentencia para ajustarnos al estandar
+        // El nodo "Description" debe aparecer antes que el nodo "ObjectIdentifier"
+//        getNode().appendChild(description);
+        
         getNode().appendChild(mimetype);
         getNode().appendChild(encoding);
     }
diff -Nru old\net\java\xades\security\xml\XAdES\ObjectIdentifierDetails.java new\net\java\xades\security\xml\XAdES\ObjectIdentifierDetails.java
--- old\net\java\xades\security\xml\XAdES\ObjectIdentifierDetails.java	Mon Jun 07 11:44:18 2010
+++ new\net\java\xades\security\xml\XAdES\ObjectIdentifierDetails.java	Tue Jun 15 11:49:14 2010
@@ -27,9 +27,9 @@
         identifier.setAttribute("Qualifier", objectIdentifier.getQualifier());
         getNode().appendChild(identifier);
 
-        Element description = createElement("Description");
-        description.setTextContent(objectIdentifier.getDescription());
-        getNode().appendChild(description);
+//        Element description = createElement("Description");
+//        description.setTextContent(objectIdentifier.getDescription());
+//        getNode().appendChild(description);
 
         if (objectIdentifier.getDocumentationReferences().size() > 0)
         {
diff -Nru old\net\java\xades\security\xml\XAdES\SignedProperties.java new\net\java\xades\security\xml\XAdES\SignedProperties.java
--- old\net\java\xades\security\xml\XAdES\SignedProperties.java	Mon Jun 07 11:44:18 2010
+++ new\net\java\xades\security\xml\XAdES\SignedProperties.java	Wed Jun 23 15:12:17 2010
@@ -1,6 +1,5 @@
 package net.java.xades.security.xml.XAdES;
 
-import javax.xml.crypto.dsig.XMLSignature;
 import org.w3c.dom.Node;
 
 /*
diff -Nru old\net\java\xades\security\xml\XAdES\XMLAdvancedSignature.java new\net\java\xades\security\xml\XAdES\XMLAdvancedSignature.java
--- old\net\java\xades\security\xml\XAdES\XMLAdvancedSignature.java	Mon Jun 14 17:42:25 2010
+++ new\net\java\xades\security\xml\XAdES\XMLAdvancedSignature.java	Wed Jun 23 15:05:39 2010
@@ -446,7 +446,7 @@
 
         List transforms = null;
         String spId = sp.getId();
-        Reference spReference = getReference(spId, transforms, xadesNamespace + "SignedProperties");
+        Reference spReference = getReference(spId, transforms, "http://uri.etsi.org/01903#" + "SignedProperties");
         referencesIdList.add(spReference);
 
         return qp;
diff -Nru old\net\java\xades\util\SystemUtils.java new\net\java\xades\util\SystemUtils.java
--- old\net\java\xades\util\SystemUtils.java	Wed Mar 17 18:18:09 2010
+++ new\net\java\xades\util\SystemUtils.java	Thu Sep 23 11:16:18 2010
@@ -35,6 +35,7 @@
 import java.util.List;
 import java.util.Properties;
 import java.util.StringTokenizer;
+import java.util.TimeZone;
 import java.util.jar.JarEntry;
 import java.util.jar.JarFile;
 import java.util.zip.ZipEntry;
@@ -74,7 +75,15 @@
             //String dateFormatString = ResourceBundleManager.getString("Default_Date_Formatter",
             //    "yyyy-MM-dd'T'HH:mm:ssZ");
             //dateFormat = new SimpleDateFormat(dateFormatString);
-            dateFormat = new ISO8601DateFormat();
+                      
+          	/*
+          	 * Modificacion insertada para uso del Cliente @firma:
+          	 * Mostramos la hora Zulu (no cambia la hora, solo el formato)
+          	 */
+          		dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+          		dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+          		
+//          		dateFormat = new ISO8601DateFormat();
         }
         return dateFormat;
     }
